queries.js

```diff
@@ -17,5 +17,8 @@ module.exports = {
     },
     getAllFavs() {
       return database('favorites');
+    },
+    updateFavorited(request) {
+      return database('books').where('isbn', request.params.isbn).update({isFavorited: request.body.isFavorited});
```

Comment:
Extreme productive learning done here.. Thank you for driving us through this Chez